app/models/facility.rb

```diff
@@ -179,9 +179,16 @@ def teleconsultation_enabled?
   end
 
   def teleconsultation_phone_number_with_isd
-    return if teleconsultation_isd_code.blank? || teleconsultation_phone_number.blank?
+    teleconsultation_phone_number = teleconsultation_phone_numbers.first
+    return if teleconsultation_phone_number.blank?
+
+    Phonelib.parse(teleconsultation_phone_number.isd_code + teleconsultation_phone_number.phone_number).full_e164
+  end
 
-    Phonelib.parse(teleconsultation_isd_code + teleconsultation_phone_number).full_e164
+  def teleconsultation_phone_numbers_with_isd
+    teleconsultation_phone_numbers.map do |phone_number|
+      { phone_number: Phonelib.parse(phone_number.isd_code + phone_number.phone_number).full_e164 }
```

Comment:
Love the `phone_number` key. Makes it easy to extend this API payload with more info later like Doctor name, serial numbers, other things.